### ISSUE TITLE
How to use it with onnx or TFlite?

### ISSUE THREAD

**Issue by vgthengane** [obss/sahi#378]
I like to thank you for open sourcing your work. 

I have successfully tried it with the `.pt` model. But I now I want to test it with other formats like `.onnx`, `.torchscript`, etc. How I can do it?

**Comment by fcakyon** [comment]
@vgthengane thanks for your interest in `SAHI`!

Which model are you talking about? YOLOv5? MMDetection? Detectron2?

**Comment by vgthengane** [comment]
Thank you for your quick reply.

I am interested in YOLOv5. Also, it is not related to `sahi` but do you think YOLOv5 will work to detect those small objects as shown in the attached image? Because even after training for 300 epochs I am not able to achieve good accuracy.
![2101220263_3](https://user-images.githubusercontent.com/39587792/153584476-245d94b4-6258-4ff6-b71a-7fed677bbf92.jpg)

**Comment by fcakyon** [comment]
@vgthengane you can use `sahi` with `.onnx` and `.torchscript` yolov5 weight files. 

You can try to slice the dataset and perform fine-tuning with that sliced dataset, then use sahi+yolov5 for increased small object detection performance :+1:
